README.md

```diff
@@ -5,4 +5,22 @@ Web app for checking tmt tests, plans and stories
 2. Install the requirements
 3. Use the `start_api.sh` script to start the api
 ## Tests
-To run the tests, use the pytest command
\ No newline at end of file
+To run the tests, use the pytest command
+
+# Requirements
+## As a user I want to have a clickable link to test/plan verifying the issue
+Covered by this [issue](https://issues.redhat.com/browse/TT-207). Implemented by creating an API, that will upon request clone the git repository, 
+parse the test/plan and return metadata in mainly machine-readable format (but a basic `html` output should
+be included as well). Test/plan is identified by fmf id given to the API as a
+url parameters. The method for attaching the link to the JIRA issue is up for discussion, but it
+should be a part of `tmt` codebase itself and would include a simple call to JIRA API.
```

Comment:
This should probably be a separate requirement:

    As a user I want to link a newly created test to a Jira issue

Reply:
Plus, perhaps, this one as well:

    As a user I want to link an existing tmt test to a Jira issue